src/Conditions/FileUploadComparator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using form_builder.Constants;
+using form_builder.Models;
+
+namespace form_builder.Conditions
+{
+    public static class FileUploadComparator
+    {
+        public static bool IsFileUploadNullOrEmpty(Condition condition, Dictionary<string, dynamic> viewModel)
+        {
+            var val = !string.IsNullOrEmpty(condition.ComparisonValue) ? bool.Parse(condition.ComparisonValue) : condition.IsNullOrEmpty;
```

Comment:
we do not need condition.IsNullOrEmpty, the ternary is there for backwards compatibility with older comparators, but this is a new one which does not need this support. can be removed.